front/app/containers/Admin/dashboard/ManagementFeed/ManagementFeedRow.tsx

```diff
@@ -34,16 +34,19 @@ const ManagementFeedRow = ({ item }: { item: ManagementFeedData }) => {
   const { data: project } = useProjectById(item.attributes.project_id);
 
   const getActionTranslation = () => {
-    switch (item.attributes.action) {
-      case 'created':
-        return formatMessage(messages.created);
-      case 'changed':
-        return formatMessage(messages.changed);
-      case 'deleted':
-        return formatMessage(messages.deleted);
-      default:
-        return item.attributes.action;
-    }
+    const action = item.attributes.action;
+
+    const actionMessages = {
```

Comment:
It would be nice to add a type to actionMessages where the key is one of the possible actions. That way if we ever add an action but miss a translation, ts will give us an error. Should look someting like: 
```const actionMessages: Record<Action, MessageDescriptor>;```